library/Notifications/Widget/ItemList/EscalationConditionListItem.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/* Icinga Notifications Web | (c) 2023 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Notifications\Widget\ItemList;
+
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\Contract\FormElement;
+use ipl\Html\FormElement\SubmitButtonElement;
+
+class EscalationConditionListItem extends BaseHtmlElement
+{
+    protected $defaultAttributes = ['class' => 'option'];
```

Comment:
`escalation-condition-list-item`

or

`escalation-condition-list > li` (in css)